delivery_service.rb

```diff
@@ -0,0 +1,40 @@
+class DeliveryService
+  attr_accessor :cars_count, :bikes_count, :auto_park
+
+  require_relative 'car'
+  require_relative 'bike'
+  require_relative 'constants'
+
+  def initialize(cars_count = 6, bikes_count = 8)
+    @auto_park = []
+    cars_count.times do
```

Comment:
Not a major offence, but it is a good code style to split your blocks `do..end` with a newline. Nothing happens if you won't do that.

Reply:
Added.Thanks